.github/workflows/stale-management.yaml

```diff
@@ -0,0 +1,22 @@
+name: Close Stale Pull Requests
+
+permissions: {}
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run every day at midnight
+
+jobs:
+  close_stale_prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close stale pull requests
+        uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This pull request has been automatically closed because it has been inactive for more than 14 days. Please reopen if you still intend to submit this pull request.'
+          # We can have different values for issues and PRs, which we might prefer
```

Comment:
Thanks for the PR @mviswanathsai ! It worth checking how Kubernetes does this.

Reply:
Hey, Shuting. @JimBugwadia also advised me to look into kubernetes. I just made this PR as a placeholder for further discussions.
Kubernetes follows the method outlined [here](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action). They run a docker image for the robot, the metrics etc. 
We could try and use the same I think, I could not find much info about it though. 
Here is the [config file](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml) they use.

Reply:
In the slack discussion, I have also linked some discussions on the Kubernetes github about the behavior of the bot itself. It might be worth checking out!

Reply:
I am not sure if we can add add `lifecycle/rotten` after `lifecycle/stale`. I will have to look into the docs again to make sure. 
Even then, this kind of behavior should be possible with adjusting the limits on the days for stale PRs and issues.

Reply:
In addition to that, we can specify labels on issues and PRs that we may not want to be automatically closed by the github action. Something which we might want to consider.

Reply:
I am working on a fork of the github stale action which will allow us to add a rotten label also. Though this might not be necessary, and I might be unsuccessful in doing so I decided to give it a shot. I hope that is not a problem!